doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -60,6 +70,9 @@ Run GeoServer from Intellij
           :width: 400
 
     #. If you want to use ``errorprone``, notably to perform the QA checks, install the ``Error Prone Compiler`` plugin, restart the IDE and set ``Javac with error-prone`` as a default compiler for the project. Please note that this will slower the build.
+#. If there are errors such as "cannot find symbol AbstractUserGroupServiceTest", rebuild the ``security-tests`` project in the security module.  Right-click on the ``security-tests`` project and click Rebuild.
+#. In the last versions of Intellij Annotations processors are enabled. If there are errors because of this uncheck this options from compiler settings.
+    .. image::img/intellij_disable_annotation_processors.jpeg
```

Comment:
This image directive is indented by one character (so it will appear as a block quote)